app/assets/javascripts/components/linkable-accordion.js

```diff
@@ -0,0 +1,22 @@
+(function(window, $) {
+  'use strict';
+
+  $(window).on('hashchange', function() {
```

Comment:
This event handler is bound to _every_ page which might not be your intention. Should this functionality be scoped based on a condition (e.g. the presence of "linkable accordion" in the markup)?